src/utils/formatTime.js

```diff
@@ -40,4 +40,13 @@ function formatCountdown (seconds) {
   }
 }
 
-export { formatDate, formatTime, formatToUTC, formatCountdown }
+function calculateTime (session) {
+  const currentTime = new Date().getTime()
+  const sessionStartTime = new Date(session.start).getTime()
+  const sessionEndTime = new Date(session.end).getTime()
+  let countdown
+  sessionStartTime > currentTime ? countdown = sessionStartTime : countdown = sessionEndTime
```

Comment:
I'm not 100% sure, but can you simplify this to `countdown = sessionStartTime > currentTime ? sessionStartTime : sessionEndTime`?